tests/integration/conftest.py

```diff
@@ -9,13 +10,22 @@
 _F = TypeVar("_F", bound=Callable)
 
 
+@dataclass(frozen=True)
+class _TestConfig:
+    data_source_config: DataSourceTestConfig
+    data: pd.DataFrame
+    extra_data: Mapping[str, pd.DataFrame]
```

Comment:
I like it 🤝 do we want to add a docstring explaining the components or is it pretty self explanatory?